web/src/localstorage.ts

```diff
@@ -1,10 +1,18 @@
+import * as z from "zod";
```

Comment:
Is there a reason to import this `as z`? We generally import things with their original name, and I'm not sure I see a reason for this to be different.

@andersk FYI in case you have thoughts on this detail.

Reply:
JavaScript modules do not have an “original name”.

However, the Zod 3 [convention](https://github.com/colinhacks/zod/#basic-usage) is `import {z} from "zod"`.